tests/components/iammeter/test_init.py

```diff
@@ -0,0 +1,17 @@
+"""Test the IamMeter component."""
+from homeassistant.components import iammeter
+
+from tests.async_mock import Mock
+
+
+async def test_setup(hass):
+    """Test setup function."""
+    config = {}
+    assert await iammeter.async_setup(hass, config) is True
```

Comment:
Just test the entry loading and unloading with `hass.config_entries.async_setup` and `hass.config_entries.async_unload`. Assert the `entry.state`.